.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build
+
+on: push
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Get Node.js version
+      id: get-node-version
+      run: echo "::set-output name=node-version::$(node --version)"
+
+    # Next step required for running actions locally via `act`
```

Comment:
⛏️ It seems odd to have CI run a step that's only required for testing CI locally.

[Looks like we can make this step conditional](https://github.com/nektos/act#skipping-steps) in `act`:

```yaml
- name: Instal Yarn
  if: ${{ env.ACT }}
  run: npm i yarn -g
```

Reply:
Agreed, love it!